core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -373,7 +373,9 @@ public void update(float delta) {
         }
         if (sneakPressed && !sneakPrevious) {
             // If player just started sneaking
-            level.makeSneak();
+            sneakPressed = level.makeSneak();
```

Comment:
Update: does not actually work, I can still sneak when ghost is here

Reply:
Yeah, I changed it so you can still sneak, but forgot to remove some of this stuff